src/Components/SmartComponents/Modals/BusinessRiskModal.test.js

```diff
@@ -15,15 +17,16 @@ describe('BusinessRiskModal', () => {
 
     it('Should render with one CVE', () => {
         const cves = [{ id: 'SOME CVE', business_risk_id: '3' }];
-        const modal = mountWithIntl(
+        render(
             <MockStore>
```

Comment:
I think we can omit using MockStore here, because we can already provide store to TestWrapper